src/main/cli/cli.c

```diff
@@ -3665,9 +3665,26 @@ static void cliGpsPassthrough(const char *cmdName, char *cmdline)
 #if defined(USE_GYRO_REGISTER_DUMP) && !defined(SIMULATOR_BUILD)
 static void cliPrintGyroRegisters(uint8_t whichSensor)
 {
-    cliPrintLinef("# WHO_AM_I    0x%X", gyroReadRegister(whichSensor, MPU_RA_WHO_AM_I));
```

Comment:
This would be much cleaner as 
```
const struct {
   const char* regName;
   uint8_t regAddr;
} gyroInfo[] = {
   {"WHO_AM_I      ", 0x72},
   {"PWR_MGMT0     ", 0x10),
...
  {NULL, 0}
};